spec/support/api_url_helper.rb

```diff
@@ -73,7 +74,29 @@ def stub_api_put(*components, body:, payload:)
     end
   end
 
-  module V2Helpers
+  module V2Expectations
+    def expect_api_v2_posts(klass, args_list, return_values = [], method: :create!)
+      # Expects the specified `method` for any class beginning with
+      # 'Sequencescape::Api::V2::' to be called with given arguments, in sequence, and returns the given values.
+      # If return_values is empty, it will return true.
+      receiving_class = "Sequencescape::Api::V2::#{klass}".constantize
+      args_list
+        .zip(return_values)
+        .each do |args, ret|
+          ret ||= true
+          expect(receiving_class).to receive(method).with(args).and_return(ret)
+        end
+    end
+
+    def expect_transfer_request_collection_creation
+      expect_api_v2_posts(
```

Comment:
Does a transfer collection creation not return anything? Just asking. Would have expected a boolean at least.

Reply:
Like the way you've set this up, to remove all the duplicated mocking all around the codebase into one place.